src/xrpld/rpc/handlers/AccountObjects.cpp

```diff
@@ -123,9 +126,18 @@ doAccountNFTs(RPC::JsonContext& context)
                 continue;
 
             if (!pastMarker && maskedNftokenID == maskedMarker &&
-                nftokenID <= marker)
+                nftokenID < marker)
                 continue;
 
+            if (nftokenID == marker)
```

Comment:
You can make the code slightly more efficient by not checking for equality if the `pastMarker` flag is set.

But once you add that condition you'll see three consecutive `if`s checking `!pastMarker`.  That's probably easier to read and maintain if it's a single condition.  So consider changing the three conditions to this:
```
            if (!pastMarker)
            {
                if (maskedNftokenID < maskedMarker)
                    continue;

                if (maskedNftokenID == maskedMarker && nftokenID < marker)
                    continue;

                if (nftokenID == marker)
                {
                    markerFound = true;
                    continue;
                }
            }
```